web/admin/application/configs/klear/model/InvoiceSchedulers.yaml

```diff
@@ -0,0 +1,135 @@
+production:
+  entity: Ivoz\Provider\Domain\Model\InvoiceScheduler\InvoiceScheduler
+  fields:
+    Iden:
+      title: _('Iden')
+      type: text
+      trim: both
+      required: true
+      maxLength: 40
+      default: true
+    frequency:
+      title: _('Frequency')
+      type: number
+      required: true
+      defaultValue: 1
+      source:
+        control: Spinner
+        min: 1
+    unit:
+      title: _('Unit')
+      type: select
+      required: true
+      source:
+        data: inline
+        values:
+        values:
+          'week': _('Week')
+          'month': _('Month')
+          'year': _('Year')
+    email:
+      title: ngettext('Email', 'Emails', 1)
+      type: text
+      trim: both
+      defaultValue: ''
+      maxLength: 140
+    taxRate:
+      title: _('Tax rate')
+      type: text
+      trim: both
+      required: true
+      sufix: "%"
+    lastExecution:
+      title: _('Last execution')
+      type: picker
+      required: true
+      source:
+        control: datetime
+        settings:
+          disabled: 'false'
+      readonly: true
+    nextExecution:
+      title: _('Next execution')
+      type: picker
+      required: true
+      source:
+        control: datetime
+        settings:
+          disabled: 'false'
+    brand:
+      title: ngettext('Brand', 'Brands', 1)
+      type: select
+      source:
+        data: mapper
+        config:
+          entity: \Ivoz\Provider\Domain\Model\Brand\Brand
+          fieldName:
+            fields:
+              - name
+            template: '%name%'
+          order:
+            Brand.name: asc
+        'null': _("Unassigned")
+    company:
+      title: ngettext('Company', 'Companies', 1)
+      type: select
+      required: true
+      source:
+        data: mapper
+        config:
+          entity: \Ivoz\Provider\Domain\Model\Company\Company
+          filterClass: IvozProvider_Klear_Filter_InvoiceSchedulerCompany
+          fieldName:
+            fields:
+              - name
+            template: '%name%'
+          order:
+            Company.name: asc
+    numberSequence:
+      title: _('Invoice number sequence')
```

Comment:
you can use _ ngettext('Invoice number sequence', 'Invoice number sequences', 0)_ here to avoid extra translate string